aci/data_source_aci_l3extsubnet.go

```diff
@@ -47,7 +41,34 @@ func dataSourceAciL3ExtSubnet() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
-		}),
+
+			"relation_l3ext_rs_subnet_to_profile": &schema.Schema{
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"tn_rtctrl_profile_name": {
+							Type:       schema.TypeString,
+							Optional:   true,
+							Deprecated: "use tn_rtctrl_profile_dn instead",
```

Comment:
It is not set so you can remove the deprecation on the datasource

Reply:
ok